docs/reference/api/docker_remote_api.md

```diff
@@ -24,6 +24,26 @@ client must have `root` access to interact with the daemon. If a group named
 `docker` exists on your system, `docker` applies ownership of the socket to the
 group.
 
+To interact with the remote API using [cURL](http://curl.haxx.se/download.html) against the unix socket, you must have 
+a version of curl greater than 7.40, after which the --unix-socket flag was added. 
+
+For example, if the Docker daemon is listening on the default socket, then you can
+use curl against the remote API to get information on running containers by:
+```
+curl --unix-socket /var/run/docker.sock http://containers/json
+```
+
+It is also possible to bind the Docker daemon to a TCP port at startup by:
```

Comment:
Maybe we should have a part for `Connecting to a remote API` (or something a little bit clearer) that detail how to connect to a remote engine, below `Using Docker Machine with the API`. And refering to it from this part.